libmscore/score.h

```diff
@@ -1086,6 +1108,7 @@ class Score : public QObject, public ScoreElement {
       virtual QVariant propertyDefault(P_ID) const override;
 
       virtual inline QQueue<MidiInputEvent>* midiInputQueue();
+      virtual inline QLinkedList<MidiInputEvent>* activeMidiPitches();
```

Comment:
Actually, I didn't realise std::list was actually a linked list too so the performance is probably roughly the same. Would you prefer a std::list?